Big-Spender/readme.md

```diff
@@ -113,6 +146,12 @@ INSERT YOUR QUERY HERE
 
 ```sql
 CREATE YOUR QUERY HERE
+
+                    SELECT date, SUM(amount) AS total_amount
+                    FROM spends
+                    WHERE date BETWEEN '2021-03-01' AND '2021-04-01'
```

Comment:
Good attempt. Note that total amounts for only those two dates are required, and not every date inbetween. Which operator will allow for multiple distinct values to be specified in your WHERE statement?